lib/history.js

```diff
@@ -440,7 +440,7 @@ async function _getAcceptableHistory({
         ({witnesses} = await _witnesses.getBlockWitnesses(
           {ledgerNode, blockHeight: localBlockHeight + 1}));
       }
-      if(witnesses.includes(creator)) {
+      if(witnesses.has(creator)) {
```

Comment:
I think it's ok to change `_witnesses.getBlockWitnesses` to return a `Set` of witnesses, +1.